appwrite/website
# Sync auth security doc: move common content to partial

## What does this PR do?

- Centralizes duplicate authentication and security documentation from `/security/authentication` and `/security` pages into the shared `auth-security.md` partial.
- Updates Markdoc configuration to properly resolve the partial fixe "partial not found" linting error

## Test Plan

- Run the app and access the URLs: `/advanced/security/authentication` and `/docs/products/auth/security`
- Confirm that there's no inconsistency/missing information
